app/[...slug]/page.tsx

```diff
@@ -27,6 +28,11 @@ const NodePage = async ({params, previewMode}: PageProps) => {
     (entity.__typename === "NodeStanfordPage" && entity.layoutSelection?.id === "stanford_basic_page_full") ||
     (entity.__typename === "NodeSulLibrary" && entity.layoutSelection?.id === "sul_library_full_width")
 
+  const sulSidebar =
+    (entity.__typename === "NodeStanfordPage" && entity.layoutSelection?.id === "sul_side_nav") ||
+    (entity.__typename === "NodeStanfordNews" && entity.layoutSelection?.id === "sul_side_nav") ||
+    (entity.__typename === "NodeSulLibrary" && entity.layoutSelection?.id === "sul_side_nav")
```

Comment:
These ID's for the news and branch pages aren't correct. Also remember to add the fields to gql queries.